operator/pkg/reconciliation/reconcile_racks.go

```diff
@@ -2112,17 +2112,6 @@ func (rc *ReconciliationContext) ReconcileAllRacks() (reconcile.Result, error) {
 		}
 	}
 
-	// TODO until we ignore status updates as it pertains to reconcile
-	// we can't switch in to a quiet period here because it will create
-	// another reconcile iteration with (likely) no work to do
-
-	// if err := rc.enableQuietPeriod(5); err != nil {
-	// 	logger.Error(
-	// 		err,
-	// 		"Error when enabling quiet period")
-	// 	return result.Error(err).Output()
-	// }
```

Comment:
Actually, Jim might want to uncomment this code now that we've fixed it in master.

Reply:
Yeah I think we should bring this back, can do it in a separate PR though.